crates/router/src/connector/rapyd/transformers.rs

```diff
@@ -550,6 +550,63 @@ impl TryFrom<types::PaymentsCancelResponseRouterData<RapydPaymentsResponse>>
     }
 }
 
+impl TryFrom<types::PaymentsSyncResponseRouterData<RapydPaymentsResponse>>
+    for types::PaymentsSyncRouterData
+{
+    type Error = error_stack::Report<errors::ParsingError>;
+    fn try_from(
+        item: types::PaymentsSyncResponseRouterData<RapydPaymentsResponse>,
+    ) -> Result<Self, Self::Error> {
+        let (status, response) = match item.response.status.status.as_str() {
+            "SUCCESS" => match item.response.data {
```

Comment:
Can we use enums instead of strings for statuses?

Reply:
this response format is from postman couldn't find in rapyd reference, we discussed in previous pr for rapyd